src/QMCWaveFunctions/Jastrow/JeeIOrbitalSoA.h

```diff
@@ -214,7 +215,7 @@ class JeeIOrbitalSoA : public WaveFunctionComponent
     }
   }
 
-  void addFunc(int iSpecies, int eSpecies1, int eSpecies2, FT* j)
+  void addFunc(int iSpecies, int eSpecies1, int eSpecies2, std::unique_ptr<FT>& j)
```

Comment:
rvalue

Reply:
After a bit study, I think value is better than rvalue to express the clear ownership.

Reply:
@ye-luo the way it's consumed right now requires an `lvalue` as we need to pass the underlying pointer to a temporary map (fcmap).

```
    auto fc = std::make_unique<FT>(*F(iG, eG1, eG2));
    fcmap[F(iG, eG1, eG2)] = fc.get();
    eeIcopy->addFunc(iG, eG1, eG2, fc);
```

Since `rvalue` is not an option we can do by value + `std::move`. What do you think?

Reply:
I don't see why rvalue is not an option. fcmap takes the value of the raw pointer and has nothing to do with smart pointers.
Yes, value + move is the preferred way.

Reply:
So fcmap will be populated from the raw pointer directly? If that's the case, rvalue is an option.